lib/Controller/Schedule.php

```diff
@@ -1208,6 +1219,13 @@ public function add(Request $request, Response $response)
             ]);
         }
 
+        if ($this->isSyncEvent($schedule->eventTypeId)) {
```

Comment:
I think this is to avoid DI of another factory - however given we've recently added Playlist/Media factories I think it makes sense to add the SyncGroupFactory here too, thereby avoiding the need for an event to load/save those items.